Readme.md

```diff
@@ -1,87 +1,153 @@
+
+SQL> SELECT A.id FROM demo_index AS A, demo_index AS B
+                    WHERE A.maxX>=B.minX AND A.minX<=B.maxX
+                    AND A.maxY>=B.minY AND A.minY<=B.maxY
+                    AND B.id=28269 ORDER BY 1;
+SELECT A.id FROM demo_index AS A, demo_index AS B WHERE A.maxX>=B.minX AND A.minX<=B.maxX AND A.maxY>=B.minY AND A.minY<=B.maxY AND B.id=28269 ORDER BY 1
+28215 28216 28262 28269
+
+SQL> quit;
+quit
+result: null
+ elapsed time: 33.303383112s
 ```
 
-Given char string `sql` as the request, the general scheme to use it is following:
- 1. `void *ptr = allocate(strlen(sql))` that returns a pointer to the memory region enough for the string
- 2. `void *res = invoke(ptr, strlen(sql))` to execute the request
- 3. read a result from the `res` by reading 4 bytes as little-endian `result_size` and the read `result_size` bytes as the final result.
- 4. `deallocate(res, strlen(sql))` to clean memory.
 
-Depends on your Wasm execution environment, `load`/`store` could be used for reading and writing a module memory.
+## Support
 
-## More insights
+Please, [file an issue](https://github.com/fluencelabs/marine/issues) if you find a bug. You can also contact us at [Discord](https://discord.com/invite/5qSnPZKh7u) or [Telegram](https://t.me/fluence_project).  We will do our best to resolve the issue ASAP.
 
-At Fluence, we use WebAssembly (Wasm) to run applications in a trustless network of independent nodes. This trustless environment mandates that every piece of the code executed by any network node must be verified by another node to check whether the execution was performed correctly. In order to be verifiable, every computation must be made deterministic, which means that to run WebAssembly code in a trustless network, we need to make its execution deterministic.
 
-There are three primary sources of nondeterminism in WebAssembly: external functions invocation, NaN payloads, and VM resource exhaustion. The first one is the most problematic; currently, we deal with it by simply prohibiting any interactions with the host environment from Wasm programs. This means that submitted WebAssembly code should not contain any external imports.
+## Contributing
 
-The main purpose of this fork is to compile to Wasm without any imports. SQlite is famous for it embeddability and allows to configure itself by variety of macros. E.g. multithreading that produces a lot of imports of host functions could be completely disabled by setting SQLITE_THREADSAFE to 0. Also this fork has been patched to use only memory database without any interaction with a hard disk. So, now it is Wasm in-memory database.
+Any interested person is welcome to contribute to the project. Please, make sure you read and follow some basic [rules](./CONTRIBUTING.md).
```

Comment:
> [rules](./CONTRIBUTING.md)
dead link

Reply:
I would say not yet alive :) 
Nice catch though. I will borrow the file from the other repo where I borrowed the text.

Reply:
Fixed